applications/callflow/test/cf_util_test.erl

```diff
@@ -16,3 +16,8 @@ alpha_to_dialpad_test() ->
     ?assertEqual(<<"23456789">>, cf_util:alpha_to_dialpad(<<"behknqux">>)),
     ?assertEqual(<<"23456789">>, cf_util:alpha_to_dialpad(<<"BeHkNqUx">>)),
     ?assertEqual(<<"23456789">>, cf_util:alpha_to_dialpad(<<"1BeH@k(N$q-u+x=">>)).
+
+alpha_to_dialpad_nonascii_test() ->
+    ?assertEqual(<<"8378">>, cf_util:alpha_to_dialpad(<<"teÕst">>)),
+    ?assertEqual(<<"8378896">>, cf_util:alpha_to_dialpad(<<"testÕÀ«©§two">>)),
+    ?assertEqual(<<"">>, cf_util:alpha_to_dialpad(<<"Ì¸ûî">>)).
```

Comment:
I've been moving more tests to use the test generator/test object style:

```erlang
alpha_to_dialpad_nonascii_test_() ->
    [?_assertEqual(<<"8378">>, cf_util:alpha_to_dialpad(<<"teÕst">>))
     ,?_assertEqual(<<"8378896">>, cf_util:alpha_to_dialpad(<<"testÕÀ«©§two">>))
     ,?_assertEqual(<<"">>, cf_util:alpha_to_dialpad(<<"Ì¸ûî">>))
    ].
```

You can then wrap these in a tuple with a human-friendly description to better indicate what was tested:

```erlang
    ,{"Test that all accented characters are stripped"
      ,?_assertEqual(<<>>, cf_util:alpha_to_dialpad(<<"Ì¸ûî">>))
    }
```

Any, just a suggestion; not required.